osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs

```diff
@@ -64,13 +64,13 @@ public override void SetUpSteps()
         {
             base.SetUpSteps();
 
-            AddStep("delete all beatmaps", () =>
+            AddStep("reset defaults", () =>
             {
                 Ruleset.Value = new OsuRuleset().RulesetInfo;
-                manager?.Delete(manager.GetAllUsableBeatmapSets());
-
                 Beatmap.SetDefault();
             });
+
+            AddStep("delete all beatmaps", () => manager?.Delete());
```

Comment:
Does this have any functional change? It doesn't seem like it would so I'm just confirming whether moving the deletion out to a separate step is nothing more than a refactoring.

Reply:
No change, just more correct.